README.md

```diff
@@ -7,10 +7,18 @@ Please fork this repository to your own account, either directly edit the README
 Your statements should include:
 
 * The problem you are addressing
+* ——The accomplishment and engagement rate for online courses are really low.
 * The eductional goal that you plan to address
+* ——My goal is to help course providers estabilish more engagable online course, through which the accomplishment can be increased. 
 * Your objectives for the project
+* ——My objective is to find the features that can improve students' engagement with online courses.
```

Comment:
This is very difficult unless you have the control to actually implement those changes. You could try to find viable candidates to test though?